recipe/meta.yaml

```diff
@@ -61,7 +59,7 @@ requirements:
   run:
     - python
     - fftw
-    - openmm 7.7
+    - openmm
     #- {{ pin_compatible('numpy') }}
```

Comment:
Think we can drop this as `numpy` doesn't appear to be needed. Would also suggest dropping a similar line above (unfortunately GH won't let me comment on that one atm)

```suggestion
```